packages/core/src/rest/api/ZosmfHeaders.ts

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
Curious is there a reason for moving this file into an "api" subdirectory of "rest"? In master branch the file is located in `packages/core/src/rest`

Reply:
no reason other than my failure at merging... fixed in commit